src/qt/bitcoingui.cpp

```diff
@@ -123,7 +123,8 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *_platformStyle, const NetworkStyle *
     spinnerFrame(0),
     platformStyle(_platformStyle)
 {
-    GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
+    QSettings settings;
+    this->restoreGeometry(settings.value("MainWindowGeometry").toByteArray());
```

Comment:
Remove this :)

Reply:
Done, and the one on line 266 as well which you missed ;)